website/app/controllers/foundations.js

```diff
@@ -0,0 +1,30 @@
+import Controller from '@ember/controller';
+
+export default class FoundationsController extends Controller {
+  items = [
+    {
+      text: 'Tokens',
+      url: 'tokens/03--how-to-use/',
```

Comment:
i think eventually we'll want to find a way to uncouple these so tightly from the specific paths like this as the names of these files seem like they could change on us?

Reply:
yes, once we finalize the information architecture and we decide which tool to use, we will also have the information about what are called the "index" pages of the components. for now this is the only way to link them

Reply:
👍  makes sense!